src/apps/concepts/components/AddConceptsIcon.tsx

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import {Link} from "react-router-dom";
+import {
+    createStyles,
+    Fab,
+    makeStyles,
+    Menu,
+    MenuItem,
+    Theme,
+    Tooltip
+} from "@material-ui/core";
+import {
+    CONCEPT_CLASSES,
+    PREFERRED_SOURCES_VIEW_ONLY,
+    useAnchor
+} from "../../../utils";
+import {Add as AddIcon} from "@material-ui/icons";
+
+interface Props {
+    canModifyDictionary: boolean;
+    canModifySource: boolean;
+    containerUrl: string;
+    gimmeAUrl: Function;
+    linkedSource?: string;
+    preferredSource: string;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        link: {
+            textDecoration: "none",
+            color: "inherit",
+            width: "100%"
+        },
+        largerTooltip: {
+            fontSize: "larger"
+        }
+    })
+);
+
+const AddConceptsIcon: React.FC<Props> = ({
+                                              canModifyDictionary,
+                                              canModifySource,
+                                              containerUrl,
+                                              gimmeAUrl,
+                                              linkedSource,
+                                              preferredSource
+                                          }) => {
```

Comment:
fix indentation

Reply:
@rakesh-1010 Its not indentation error but you want this to be consistent across all the files, we can make the change